RedDeadOxygen/Assets/Scripts/Instances/Player.cs

```diff
@@ -32,10 +32,22 @@ public class Player : MonoBehaviour
     private Timer _speedTimer;
     private List<Ressource> _Ressources = new List<Ressource>();
 
+    private MovementCategory _movementCategory = MovementCategory.e_ForceBased;
+    private float _wallElasticity = 0f;
+
+    Vector2 movement, movementOrder;
+    float _maxForceSpeed = 4.0f, _currentMaxForceSpeed = 4.0f;
```

Comment:
Declare variable on multi-lines

```suggestion
    float _maxForceSpeed = 4.0f, 
    float _currentMaxForceSpeed = 4.0f;
```